### ISSUE TITLE
JSONSchema for Data

### ISSUE THREAD

**Issue by kaplun** [inspirehep/inspire-next#1618]
We need to map Data to JSONSchema 
* [ ] design proper JSONSchema (possibly compatible with hep.json)
* [ ] implement dojson rules for porting from legacy.

**Comment by jacquerie** [comment]
I think on friday we agreed to wait for @suenjedt to get back to us on this, no?

**Comment by suenjedt** [comment]
Yes, @ArtemisLav can update you on what we are doing.

**Comment by ArtemisLav** [comment]
So, we have started developing a basic data model for the Open Data Portal using schema.org for the data schema and JSON-LD for the syntax. What I have so far is here: https://github.com/ArtemisLav/DataModel_JSON-LD-schema.org/blob/master/ODP 
Any feedback is more than welcome.

**Comment by kaplun** [comment]
Thanks @ArtemisLav. This on the surface looks totally different from the JSONSchemas we have built for INSPIRE so far (https://github.com/inspirehep/inspire-schemas/tree/master/inspire_schemas/records) but I guess we should see this work as source of information for properly designing the internal JSONSchemas, right?

IRL meeting will surely clarify everything :smile:

**Comment by ArtemisLav** [comment]
Yes, they do appear to be quite different, but what I showed you is in the initial stages of development; it was just an attempt to see how it would look like if we were to implement it. 

I would be happy to chat with you about this.

**Comment by ArtemisLav** [comment]
In preparation for our meeting tomorrow, I have encoded one of the INSPIRE data records in the JSON-lD/schema.org format, so that you can take a look:
https://gist.github.com/ArtemisLav/b8a6cb19e6a4f0242a309657633dff49

**Comment by jacquerie** [comment]
Things we learned:

* What @ArtemisLav is doing can't be directly reused by us, but it can inform the writing of our schemas for data.
* We should start using `schema.org` terminology in our JSON schemas as much as possible, starting from data.
* Eventually, we should consume JSON-LD from OpenData and HEPData.
* Eventually, we should expose JSON-LD (from our schemas + a JSON-LD context).
